packages/address-utils/tsup.config.ts

```diff
@@ -6,6 +6,6 @@ export default defineConfig({
   dts: true,
   clean: true,
   outExtension: ({ format }) => ({
-    js: format === "esm" ? ".mjs" : ".cjs",
+    js: format === "esm" ? ".js" : ".cjs",
```

Comment:
Since we use tsc in dev mode and it outputs `.js` files, and `package.json` lists those `.js` files for the `import` option of a module, I updated the production build to output `.js` files for esm format.